.github/workflows/main.yml

```diff
@@ -0,0 +1,9 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
can the initial workflow run compilation?

Reply:
Unfortunately, GitHub seems to require a workflow to be checked into the main branch before it will run for pull requests, so this is an essentially empty workflow that allows testing the real initial one.